### PR TITLE
more fixes, again

### PR DESCRIPTION
**Pull request by xuzhen**

### CODE REVIEW

**Comment by M7S** [comment]
Previews! That is great!

**Comment by M7S** [comment]
I get the following error:
  File "dockx", line 1503, in <module>
    dockx = DockX(monitor)
  File "dockx", line 496, in __init__
    if Gdk.MAJOR_VERSION > 3 or Gdk.MINOR_VERSION >= 22:
  File "/usr/lib/python3/dist-packages/gi/overrides/__init__.py", line 39, in __getattr__
    return getattr(self._introspection_module, name)
  File "/usr/lib/python3/dist-packages/gi/module.py", line 137, in __getattr__
    self.__name__, name))
AttributeError: 'gi.repository.Gdk' object has no attribute 'MAJOR_VERSION'

Should it be Gtk.MAJOR_VERSION?

**Comment by xuzhen** [comment]
It seems that only the new version of GDK has this. I'll change it.

File: dockbarx/groupbutton.py [hunk 1]
```diff
@@ -2203,6 +2202,8 @@ def show(self, delay=None, force=False):
         except:
             logger.exception("If an empty popup was shown this " + \
                              "might have somethin to do with it:")
+        for window in group:
+            window.item.set_show_preview(True)
```
Comment (M7S) [inline dockbarx/groupbutton.py:2206]: What does these lines do? They makes dockbarx ignore the option "Show previews" in preferences dialog and always shows the previews no matter what, but do they serve any other purpose?

Comment (M7S) [reply dockbarx/groupbutton.py:2206]: Removing them seems to make the option "Show previews" work again but I don't want to remove the lines if the serve another purpose as well.

Comment (xuzhen) [reply dockbarx/groupbutton.py:2206]: Nice catch, thanks! I'll fix it in next pull request.

**Pull request merged**